terraform/modules/neptune/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  neptune_port             = 8182
+  protocol                 =  "tcp"
+  allowed_ip_blocks        = ["129.43.0.0/16", "137.187.0.0/16", "10.128.0.0/9", "165.112.0.0/16", "156.40.0.0/16", "10.208.0.0/21", "128.231.0.0/16", "130.14.0.0/16", "157.98.0.0/16", "10.133.0.0/16"]
```

Comment:
We need to talk to Mike about this - because data commons is a public repository, we probably shouldn't put ranges in the repository.